### PR TITLE
fix(render): create svg elements with the right namespace

### PR DESCRIPTION
**Pull request by tbosch**
Temporary fix for #4506

### CODE REVIEW

File: modules/angular2/src/core/dom/abstract_html_adapter.dart [hunk 1]
```diff
@@ -215,6 +215,10 @@ abstract class AbstractHtml5LibAdapter implements DomAdapter {
     return new Element.tag(tagName);
   }
 
+  createElementNS(ns, tagName, [doc]) {
+    throw 'not implemented';
```
Comment (vicb) [inline modules/angular2/src/core/dom/abstract_html_adapter.dart]: the class is abstract, no need to throw (other places should also be modified)

File: modules/angular2/src/core/render/dom/dom_renderer.ts [hunk 2]
```diff
@@ -32,6 +32,92 @@ import {createRenderView, NodeFactory} from '../view_factory';
 import {DefaultRenderView, DefaultRenderFragmentRef, DefaultProtoViewRef} from '../view';
 import {camelCaseToDashCase} from './util';
 
+// TODO(tbosch): solve SVG properly once https://github.com/angular/angular/issues/4417 is done
+const XLINK_NAMESPACE = 'http://www.w3.org/1999/xlink';
+const SVG_NAMESPACE = 'http://www.w3.org/2000/svg';
+const SVG_ELEMENT_NAMES = CONST_EXPR({
```
Comment (vicb) [inline modules/angular2/src/core/render/dom/dom_renderer.ts]: nit: ` SVG_ELEMENTS`  ?

File: modules/angular2/src/core/dom/abstract_html_adapter.dart (hunk 1 above)
Comment (yjbanov) [inline modules/angular2/src/core/dom/abstract_html_adapter.dart]: @vicb: could you please file an issue for this? There are lots of places where we need to clean this up. For now Tobias' change is fine. The P1 goal is to roll out pre-compiler.

Comment (tbosch) [inline modules/angular2/src/core/dom/abstract_html_adapter.dart]: But then I need to implement it in all sub classes...

**Pull request closed without merging**